controllers/nodenetworkconfigurationpolicy_utils.go

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
Could you please move this file to `pkg/enactment/`? That should be a better home for it.
Also, a better name would be nice, `message.go` maybe?